internal/api/api.go

```diff
@@ -63,64 +63,70 @@ func (api CertAPIClient) IsOperatorCertified(org, packageName string) bool {
 }
 
 // GetImageByID get container image data for the given container Id
-func (api CertAPIClient) GetImageByID(id string) (response string, err error) {
-	var responseData []byte
+func (api CertAPIClient) GetImageByID(id string) (string, error) {
```

Comment:
My opinion on this is it's good to use in functions with few lines of code such as this one where the logic is not complicated and you can think of that name in signature as the declaration of variable itself. 

Is it recommended not to use this future altogether or only when the length/complexity of a function is "significant" ?

Reply:
The output variable names are actually a good way to auto-document them. So, in case they are removed, I think we should document them in the function's comment.

Reply:
I like the idea of removing naked returns, but as @greyerof mentioned it would be nice to document the function ouptut then.

Reply:
Sounds good, I'll add the output in the comment.